app/models/training_data/code_tags_sample.rb

```diff
@@ -70,4 +70,15 @@ def next_status
       status
     end
   end
+
+  def self.sample_status(filter_status)
```

Comment:
@iHiD I couldn't come up with a good name. Let me know if you have any suggestions. I also considered adding a scope, but that would get messy I think

Reply:
`statuses_for_filter`?